tests/br_gcs/run.sh

```diff
@@ -19,10 +19,11 @@ TABLE="usertable"
 DB_COUNT=3
 
 GCS_HOST="localhost"
-GCS_PORT=21808
+GCS_PORT=$(shuf -i 21808-31808 -n1) # Generate a random port.
```

Comment:
I'm a bit confused what's the different between this random port with the previous random port `21808`